test/Cover/CoverGeneratorFactoryTest.php

```diff
@@ -31,15 +31,20 @@
 
 namespace OpusTest\Pdf\Cover;
 
-use Opus\Pdf\Cover\CoverGeneratorFactory;
-use Opus\Pdf\Cover\CoverGeneratorInterface;
+use Opus\Common\Config;
+use Opus\Common\Cover\CoverGeneratorFactory;
+use Opus\Common\Cover\CoverGeneratorInterface;
+use Opus\Pdf\Cover\DefaultCoverGenerator;
 use PHPUnit\Framework\TestCase;
 
 class CoverGeneratorFactoryTest extends TestCase
 {
     public function testCreate()
     {
-        $generator = CoverGeneratorFactory::create();
+        $generatorClass = DefaultCoverGenerator::class;
+        Config::setValueInConfig(Config::get(), 'pdf.covers.generatorClass', $generatorClass);
```

Comment:
Die Funktion `setValueInConfig` gibt es nur, weil es sich nicht vermeiden lies für das Editieren der Konfiguration in der Administration. Bitte nicht in Tests verwenden. Für einen UNIT Test im Idealfall die notwendige Konfiguration direkt in die Klasse injizieren, anstatt über die globalen Konfiguration zu gehen.